project-koku/koku
# COST-2106: Remove unused tables from UI summary list.

Changes needed prior to running migrations for  [COST-2106](https://issues.redhat.com/browse/COST-2106)

Changes proposed in this PR:
* Remove unused UI tables from the list prior to running the migrations to prevent chicken and egg problem.

Testing Instructions:
Upload gcp data and ensure those tables don't get called during the UI_SUMMARY_TABLES resummary is called. 

- These changes probably should of been in https://github.com/project-koku/koku/pull/3400
